<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelMatcherComponent</class>
 <widget class="QFrame" name="CModelMatcherComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gl_ModelMatcher">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="1">
    <widget class="swift::gui::components::CAircraftCombinedTypeSelector" name="comp_CombinedCode">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lbl_Livery">
     <property name="text">
      <string>Livery:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="swift::gui::components::CDbAircraftIcaoSelectorComponent" name="comp_AircraftSelector">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QFrame" name="fr_Buttons">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="hs_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Settings">
        <property name="text">
         <string>settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_ReverseLookup">
        <property name="text">
         <string>reverse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_ModelMatching">
        <property name="text">
         <string>matching</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_withReverseLookup">
        <property name="toolTip">
         <string>use reverse lookup for model matching</string>
        </property>
        <property name="text">
         <string>with reverse lookup</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_Manufacturer">
     <property name="placeholderText">
      <string>e.g. Cessna</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QFrame" name="fr_Results">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="vl_Results" stretch="1,3">
      <item>
       <widget class="QTextEdit" name="te_Results">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="swift::gui::views::CStatusMessageView" name="tvp_ResultMessages">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_Manufacturer">
     <property name="text">
      <string>Manufacturer:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="le_ModelString">
     <property name="placeholderText">
      <string>model string</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_Aircraft">
     <property name="text">
      <string>Aircraft:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbl_Model">
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lbl_Airline">
     <property name="text">
      <string>Airline:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbl_ModelSetCount">
     <property name="text">
      <string>Model set:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="le_Callsign">
     <property name="placeholderText">
      <string>e.g. DAMBZ</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_Simulator">
     <property name="text">
      <string>Simulator:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="swift::gui::components::CDbLiverySelectorComponent" name="comp_LiverySelector">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignLeft">
    <widget class="swift::gui::components::CSimulatorSelector" name="comp_SimulatorSelector">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QFrame" name="fr_ModelSet">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="hl_ModelSet">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="le_ModelSetCount">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_UseWorkbench">
        <property name="text">
         <string>use workbench</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="swift::gui::components::CDbAirlineIcaoSelectorComponent" name="comp_AirlineSelector">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_Combined">
     <property name="text">
      <string>Combined:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lbl_Callsign">
     <property name="text">
      <string>Callsign:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDbAirlineIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/dbairlineicaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDbLiverySelectorComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/dbliveryselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CAircraftCombinedTypeSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/aircraftcombinedtypeselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CDbAircraftIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/dbaircrafticaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::views::CStatusMessageView</class>
   <extends>QTableView</extends>
   <header>gui/views/statusmessageview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comp_SimulatorSelector</tabstop>
  <tabstop>le_ModelSetCount</tabstop>
  <tabstop>cb_UseWorkbench</tabstop>
  <tabstop>comp_AircraftSelector</tabstop>
  <tabstop>comp_AirlineSelector</tabstop>
  <tabstop>comp_CombinedCode</tabstop>
  <tabstop>comp_LiverySelector</tabstop>
  <tabstop>le_Manufacturer</tabstop>
  <tabstop>le_Callsign</tabstop>
  <tabstop>le_ModelString</tabstop>
  <tabstop>pb_Settings</tabstop>
  <tabstop>pb_ReverseLookup</tabstop>
  <tabstop>pb_ModelMatching</tabstop>
  <tabstop>cb_withReverseLookup</tabstop>
  <tabstop>te_Results</tabstop>
  <tabstop>tvp_ResultMessages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
